libres/lib/analysis/ies/ies_enkf_data.cpp

```diff
@@ -221,45 +202,26 @@ void ies_enkf_data_augment_initialE(ies_enkf_data_type *data,
                 m++;
             }
         }
-
-        if (dbg) {
-            int m_nrobs = std::min(obs_size_msk - 1, 50);
-            int m_ens_size = std::min(ens_size_msk - 1, 16);
-            matrix_pretty_fprint_submat(data->E, "data->E", "%11.5f",
-                                        data->log_fp, 0, m_nrobs, 0,
-                                        m_ens_size);
-        }
     }
 }
 
 void ies_enkf_data_store_initialA(ies_enkf_data_type *data,
                                   const matrix_type *A) {
     if (!data->A0) {
         // We store the initial ensemble to use it in final update equation                     (Line 11)
-        bool dbg = ies_enkf_config_get_ies_debug(data->config);
         int m_state_size = std::min(matrix_get_rows(A) - 1, 50);
         int m_ens_size = std::min(matrix_get_columns(A) - 1, 16);
-        fprintf(data->log_fp, "Allocating and assigning data->A0 \n");
         data->A0 = matrix_alloc_copy(A);
-        if (dbg)
-            matrix_pretty_fprint_submat(data->A0, "Ini data->A0", "%11.5f",
-                                        data->log_fp, 0, m_state_size, 0,
-                                        m_ens_size);
     }
 }
 
-void ies_enkf_data_allocateW(ies_enkf_data_type *data, int ens_size) {
+void ies_enkf_data_allocateW(ies_enkf_data_type *data) {
     if (!data->W) {
-        // We initialize data-W which will store W for use in next iteration                    (Line 9)
-        bool dbg = ies_enkf_config_get_ies_debug(data->config);
+        // We initialize data->W which will store W for use in next iteration                    (Line 9)
+        int ens_size = bool_vector_size(data->ens_mask);
         int m_ens_size = std::min(ens_size - 1, 16);
```

Comment:
with `dbg` removed, `m_ens_size` is unused.

Reply:
Ok